app/models/planet.py

```diff
@@ -0,0 +1,7 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
```

Comment:
Careful! Using the class as it currently is, we'd be able to make a planet with a null name, description, and color. I do see in the `POST` request that y'all checked for falsy values for name, description, and color, however, null values can still be inserted manually using SQL. When making classes in the future, consider which columns are required (non-nullable).

Reply:
Also, I highly recommend adding a helper function to turn planet instances into a dictionary (this will definitely save you from writing a few lines of code down the road)!